networking/_posts/2017-01-15-how-to-configure-a-custom-subdomain.md

```diff
@@ -8,13 +8,15 @@ description: How to configure a Custom Subdomain
 Runnable supports routing subdomain requests to your containers.
```

Comment:
This comment is not specific to this line, but can we make the headline for this article title case?